[WIP] Chapter 1 revision: reproducibility focus

bbdaniels opened the pull request:
@mariaruth – this is not totally complete but it is ready for your review. I reworked the structure and added a bit of new content. I am still missing a bit at the end that the skeleton flagged for "what DIME does" as I don't think we have an overall structure for what that should look like? but the rest is ready for your input. PDF attached here.

[Ch1 BBD.pdf](https://github.com/worldbank/d4di/files/4618727/Ch1.BBD.pdf)

mariaruth commented:
This is great. I added comments in the pdf, mainly on structure, and suggestions for how to incorporate what DIME does in the text. happy to chat about any of it if comments are not clear or you want to bounce ideas around. 
[Ch1.BBD.MRJ.pdf](https://github.com/worldbank/d4di/files/4644779/Ch1.BBD.MRJ.pdf)

bbdaniels commented:
Addressed all comments except adding DIME examples. I also took the opportunity to reorg a little so the sections get broadly keyed to the three themes. Let me know if this works or if it seems clumsy. Thanks!

[ch1-revised.pdf](https://github.com/worldbank/d4di/files/4726795/ch1-revised.pdf)
